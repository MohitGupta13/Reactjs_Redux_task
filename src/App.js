import React, {Component} from 'react';
import './App.css';
import {Provider} from "react-redux";
import {store} from "./redux/store";
import HobbySelectorCheck from "./components/HobbySelectorCheck";
/*import CustomerList from "./components/CustomerList";*/
let App = () => {

    return(
        <React.Fragment>
            <nav className="navbar navbar-dark bg-dark navbar-expand-sm">
                <a href="/" className="navbar-brand">React Redux -Checkbox Application</a>
            </nav>
            <Provider store={store}>
                <HobbySelectorCheck/>
            </Provider>

         <div style={{marginBottom : '200px'}}/>
        </React.Fragment>
    );
};
export default App;
