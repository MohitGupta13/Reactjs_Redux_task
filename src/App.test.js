import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

test("snapshot Test",()=> {
const tree =renderer.create(<App/>).toJSON();
console.log(tree);
expect(tree).toMatchSnapshot();
})