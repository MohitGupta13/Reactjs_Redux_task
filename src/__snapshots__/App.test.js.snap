// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Test 1`] = `
Array [
  <nav
    className="navbar navbar-dark bg-dark navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      React Redux -Checkbox Application
    </a>
  </nav>,
  <div
    className="container mt-3"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8"
      >
        <div
          className="card"
        >
          <div
            className="card-header bg-secondary text-white"
          >
            <p
              className="h4"
            >
              Hobby Selector
            </p>
          </div>
          <div
            className="card-body"
          >
            <div
              className="row"
            >
              <div
                className="col-md-4"
              >
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    id="defaultCheck1"
                    name="eating"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="defaultCheck1"
                  >
                    Eating
                  </label>
                </div>
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    id="defaultCheck2"
                    name="coding"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="defaultCheck2"
                  >
                    Coding
                  </label>
                </div>
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    id="defaultCheck3"
                    name="sleeping"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="defaultCheck3"
                  >
                    Sleeping
                  </label>
                </div>
              </div>
              <div
                className="col-md-8"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    style={
      Object {
        "marginBottom": "200px",
      }
    }
  />,
]
`;
